web/src/components/tasks/business-formation/contacts/ContactsStep.tsx

```diff
@@ -38,22 +38,30 @@ export const ContactsStep = (): ReactElement => {
     return (Config.formation.fields as any)[field].description;
   };
 
+  const isCorpAndLp = (): boolean => {
+    return [...corpLegalStructures, "limited-partnership"].includes(state.formationFormData.legalType);
+  };
+
+  const isCorp = (): boolean => {
+    return [...corpLegalStructures].includes(state.formationFormData.legalType);
+  };
+
   return (
     <>
       <div data-testid="contacts-step">
         <RegisteredAgent />
         {shouldShowMembers() && (
           <>
             <hr className="margin-top-0 margin-bottom-3" />
-            {doesFieldHaveError("members") && (
+            {doesFieldHaveError("members") && !isCorp() && (
```

Comment:
The !isCorp condition is necessary so that this alert doesn't render for domestic corporations (s & c), otherwise, the banner alert will populate and we now want an inline instead for domestic corporations.